src/ContinuumAbsorption/absorption_ff_neutral_metals_molecules.jl

```diff
@@ -0,0 +1,422 @@
+const _long_wavelength_ff_interps = let
+    T_vals = SA[ 50.0,    100.0,    500.0,    1000.0,   5000.0,   10000.0,   25000.0]
+
+    table = Dict(
+        "He"  => [    1.34,     0.968,    0.475,     0.353,    0.173,     0.13,      0.07  ],
+        "Ne"  => [    0.107,    0.0823,   0.0604,    0.056,    0.041,     0.0328],  #-----
+        "Ar"  => [    1.4,      0.673,    0.057,     0.019,    0.038,     0.062,     0.097 ],
+        "Kr"  => [    6.17,     3.37,     0.527,     0.17,     0.034,     0.073,     0.138 ],
+        "Xe"  => [   27.7,     13.8,      1.74,      0.548,    0.127,     0.234,     0.303 ],
+        "Li"  => [   20.5,     41.6,     60.0,      28.0,      0.93,      0.062,     0.012 ],
+        "Na"  => [   18.6,     24.6,     47.0,      26.0,      2.7,       1.3],     #-----
+        "Cs"  => [ 1695.0,    853.0,     88.0,      57.0,      4.7,       2.8,       1.2   ],
+        "Hg"  => [    5.09,     3.02,     0.845,     1.54],   #-----      -----      -----
+        "N"   => [    0.263,    0.195,    0.13,      0.12,     0.1,       0.0904,    0.0699],
+        "O"   => [    0.280,    0.202,    0.119,     0.103,    0.0743,    0.0654,    0.0512],
+        "H2"  => [    1.9,      1.47,     0.866,     0.703,    0.429,     0.29,      0.112 ],
+        "N2"  => [    0.597,    0.554,    0.506,     0.459,    0.333,     0.293,     0.14  ],
+        "O2"  => [    0.762,    0.54,     0.343,     0.261,    0.171,     0.114,     0.076 ],
+        "CO"  => [   21.3,      1.14,     0.642,     0.605,    0.45,      0.28,      0.15  ],
+        "CO2" => [   43.5,     21.7,      4.1,       1.75,     0.192,     0.16,      0.13  ],
+        "H20" => [  996.0,    342.0,     28.0]      #-----     -----      -----      -----
+    )
+
+    Dict(k => LinearInterpolation(view(T_vals, 1:length(v)), v.*1e-34, extrapolation_bc = Throw())
+         for (k,v) in table)
+end
+
+
+# load free-free absorption data from John 1975, MNRAS, 172, 305. This data is valid for
+# 0.5 μm ≤ λ ≤ 10 μm. Most of these tables share a common set of λ and T values.
+const _typical_john75b_wavelengths = SA[1.0e-3, 5.0e-4, 2.5e-4, 1.5e-4, 1.0e-4, 7.5e-5, 5.0e-5] #cm
+const _typical_john75b_T_vals = SA[100.0, 500.0, 1000.0, 2500.0, 5000.0, 7500.0, 10000.0, 12500.0,
+                                   15000.0] #K
+
+function _build_interpolator_john75b(λ_vals, T_vals, elements)
+    LinearInterpolation((reverse(λ_vals), T_vals),
+                        view(elements,size(elements)[1]:-1:1,:), extrapolation_bc=0)
+end
+
+const _short_wavelength_Heminus_ff_interp = let
+    # this comes from Table I of John 1975, MNRAS, 172, 305 and it holds He⁻ ff absorption
+    # cross sections (including the correction for stimulated emission)
+    #
+    # we don't plan to use this, particular data (we have more modern data for He⁻ ff absorption),
+    # this is primarily for testing
+    elements = SMatrix{7,9}(
```

Comment:
Given the discussion about stack space we had, do you think we should use static arrays here?  (I genuinely don't know.)